run.py

```diff
@@ -51,8 +69,8 @@ def process(message,username,chatid):
 			sendMsg("@"+username+" you are already registered.",chatid)
 	elif "/balance" in message[0]:
 		try:
-			(balance, pending_balance) = returnBal(username)
-			sendMsg("@"+username+" Balance : "+balance+ "Doge ("+pending_balance+" Doge)",chatid)
+			(balance, pending_balance, balance_msg, pending_msg) = returnBal()
+			sendMsg("@"+'vindard'+balance_msg+pending_msg,chatid)
```

Comment:
@vindard i think this was left behind from testing - it will return your username each time `/balance` is sent.

Reply:
Ah good catch. Just fixed.